marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/WebSecurityConfiguration.java

```diff
@@ -55,12 +59,19 @@ protected void configure(HttpSecurity http) throws Exception {
             .and()
             // Define requests that are always permitted, regardless of whether the user is authenticated or not
             .authorizeRequests()
+                // Needed for websocket tests
+//                .antMatchers("/websocket/**").permitAll()
                 // Needed for springfox to work - see https://github.com/springfox/springfox/issues/1996#issuecomment-335155187
                 .antMatchers("/swagger-resources/**", "/swagger-ui.html", "/v2/api-docs", "/webjars/**").permitAll()
                 // Non-springfox patterns to permit
-                .antMatchers(getAlwaysPermittedPatterns()).permitAll().anyRequest().authenticated()
-            .and()
-            .logout().logoutUrl("/api/logout").logoutSuccessHandler(((request, response, authentication) -> request.getSession().invalidate()));
+                .antMatchers(getAlwaysPermittedPatterns()).permitAll();
+        // needed for WebSocket test
+        if (environment.getProperty("hub.websocket.securityDisabled","false").equals("true")) {
+            http.authorizeRequests().antMatchers("/websocket/**").permitAll();
+        }
```

Comment:
@rjrudin, FYI, we are permitting access to ```/websocket/**``` only while running tests.  So the production app has all endpoints behind spring security.